scripts/main/designer/components/inspector/inspector-component/Fields/Number/inspector-number-component.directive.js

```diff
@@ -0,0 +1,7 @@
+angular.module('designer.module')
+    .directive('numberField', function () {
+        return {
+            restrict: 'E',
+            templateUrl: 'main/designer/components/inspector/inspector-component/Fields/Number/inspector-number-component.template.html'
```

Comment:
don't use camel case for file and folder names

Reply:
done